content/en/docs/architecture/release-gating.md

```diff
@@ -137,4 +140,4 @@ After at least 2 sprints of informing status, you may request the job become blo
 
 Blocking jobs come with the responsibility of being responsive to breakages.  Generally, TRT expects a team that requests
 a blocking job to respond to a breakage within 4 hours during their normal working hours only. In exchange for this agreement,
-TRT will monitor the jobs, alert on them, and revert changes in the product that caused the breakage.
\ No newline at end of file
+TRT will monitor the jobs, alert on them, and revert changes in the product that caused the breakage.
```

Comment:
lets add an empty line at the end of the file

Reply:
It doesn't show up in GitHub but it's there:
```
$ git cat-file --textconv refs/heads/analysis:content/en/docs/architecture/release-gating.md | tail -1 | xxd -s +96
00000060: 6b61 6765 2e0a                           kage..
```